src/libraries/Microsoft.Extensions.Configuration.FileExtensions/src/FileConfigurationProvider.cs

```diff
@@ -34,7 +34,8 @@ public FileConfigurationProvider(FileConfigurationSource source!!)
                     () =>
                     {
                         Thread.Sleep(Source.ReloadDelay);
-                        Load(reload: true);
+                        throw new NotImplementedException("Need to know the separator string here...");
```

Comment:
This seems out of place

Reply:
Oops! Sorry, I'll look at that - must've overlooked it as I was tidying up